pkg/cmd/server/origin/master.go

```diff
@@ -1063,6 +1067,58 @@ func (c *MasterConfig) RunSecurityAllocationController() {
 	controller.Run()
 }
 
+func (c *MasterConfig) ensureDefaultSecurityContextConstraints() {
+	sccList, err := c.KubeClient().SecurityContextConstraints().List(labels.Everything(), fields.Everything())
+	if err != nil {
+		glog.Fatalf("Unable to initialize security context constraints: %v", err)
+	}
+	if len(sccList.Items) > 0 {
+		return
+	}
+	glog.Infof("No security context constraints detected, adding defaults")
+	sccAdmin := &kapi.SecurityContextConstraints{
```

Comment:
define the bootstrap SCC's, then iterate over them and do the create. errors probably shouldn't terminate the server, just log as errors, since the cluster admin can create them in a running server